scripts/Util.py

```diff
@@ -4316,14 +4317,11 @@ def getCommonDir(self, current):
         return os.path.join(self.getPath(), "test", "Common")
 
     def getCommandLine(self, current, process, exe, args):
-        return "node {0}/run.js {1} {2}".format(self.getCommonDir(current), exe, args)
-
-    def getEnv(self, process, current):
-        env = Mapping.getEnv(self, process, current)
-        env["NODE_PATH"] = os.pathsep.join(
```

Comment:
NODE_PATH is not used to resolve imports from ES modules, https://nodejs.org/api/esm.html#esm_no_node_path